modules/update-mirror-repository.adoc

```diff
@@ -3,10 +3,8 @@
 // * updating/updating-restricted-network-cluster.adoc
 
 :_content-type: PROCEDURE
-[id="update-mirror-repository_{context}"]
```

Comment:
nit: should we preserve this old anchor to avoid breaking any previous linkers?  Or shift this anchor to a new location that mentions both the oc-mirror and `oc adm release mirror` options?

Reply:
This id has been removed from this module and added to the main "Mirroring the OpenShift Container Platform image repository" title in the assembly itself. So the link should still exist and work as before, and it should behave exactly the way you suggest in your second sentence.

However, I forgot to do the same thing for the OSUS section of this page, so I'll do that now. Thanks!